src/scss/styles.scss

```diff
@@ -188,12 +188,25 @@ input, select, textarea {
 }
 
 .container {
-    width: 980px;
+    min-width: 980px;
+    width: 90%;
     max-width: none !important;
     margin: 0 auto;
     padding: 0;
+    .row {
```

Comment:
I would think this should be removed... leaving the `.col-*` classes as-is should still allow full-width display to work w/o wrapping and we would want scaling to work across the board (not just for the center region).